test/e2e/handler/bonding_default_interface_test.go

```diff
@@ -22,7 +22,7 @@ func boundUpWithPrimaryAndSecondary(bondName string) nmstate.State {
     link-aggregation:
       mode: active-backup
       options:
-        miimon: '140'
+        miimon: 140
```

Comment:
What about https://github.com/nmstate/kubernetes-nmstate/blob/master/docs/examples/bond.yaml?

Is this backwards compatible?

Reply:
Yep, test is working fine, the issue is with NNSs, since it reports "port" instead of "slaves".